src/fdk_fulltext_search/search/query_utils.py

```diff
@@ -1,141 +1,98 @@
 import re
 
-from fdk_fulltext_search.ingest.utils import IndicesKey
-from fdk_fulltext_search.search.fields import (
-    index_description_fields,
-    index_fulltext_fields,
-    index_title_fields,
-)
-from fdk_fulltext_search.search.query_utils_dataset import (
-    autorativ_dataset_query,
-    open_data_query,
-)
 
+def autorativ_data_service_query() -> dict:
+    return {"term": {"nationalComponent": "true"}}
 
-def title_term_query(field, search_string):
-    return {"term": {field: search_string}}
 
+def autorativ_dataset_query() -> dict:
```

Comment:
Denne flyttes fra en spesifikk datasett-query-utils til en felles? Men den er vel ganske spesifikk for datasett, eller kan den brukes på tvers?

Reply:
Jeg er flyttet til filter og agg utils. Denne kan brukes på tvers. Siden det er should så gjør det ingenting når den ikke får treff.